configure/CONFIG_COMMON

```diff
@@ -38,8 +38,6 @@ BUILD_ARCHS = $(EPICS_HOST_ARCH) $(CROSS1) $(CROSS2)
 # otherwise override this in os/CONFIG_SITE.<host_arch>.Common
 PERL = perl -CSD
 
-PYTHON = python
```

Comment:
Removing `$(PYTHON)` would break the `sphinx` target, not to mention dependent modules.  So this has to stay.

https://github.com/epics-base/epics-base/blob/e4ad4becde5ebe76b2251bef0fdfccd4fd1101ea/documentation/Makefile#L169-L171

At some point, perhaps now, it would make sense to switch the default to `PYTHON=python3`.

fyi. Debian derivatives have `apt-get install python-is-python3` to set a symlink `/usr/bin/python` -> `python3`.

Reply:
Restored `$(PYTHON)` which is now set to `python3`, with adjusted comments.
Recent macOS releases don't ship with any python, although it is easily obtained.

This change needs a Release Notes entry, which I have not included yet.